csvcubed/csvcubed/utils/validators/uri.py

```diff
@@ -23,6 +23,9 @@ class SomeModel(PydanticModel):
 
             _some_uri_validator = validate_uri("some_uri")
     """
-    return validator(attr_name, allow_reuse=True, always=not is_optional)(
-        ensure_looks_like_uri
-    )
+    # attr_name = list(attr_name)
+    # attr_name = list(attr_name.split(" "))
+    for name in attr_name:
+        return validator(name, allow_reuse=True, always=not is_optional)(
```

Comment:
This won't work as expected. The function `validate_uri` function should always take a single `attr_name` since it is the name of the attribute to be inspected inside the model. It returns a validator which can be asked to validate the contents of `model.attr_name` at a later point in time.